java-frontend/src/main/java/org/sonar/java/se/xproc/BehaviorCache.java

```diff
@@ -122,9 +116,6 @@ public MethodBehavior get(String signature) {
 
   @CheckForNull
   private MethodBehavior get(String signature, @Nullable Symbol.MethodSymbol symbol) {
-    if (isknownSignature(BLACKLIST, signature)) {
```

Comment:
Removing this should probably lead to simplification of the isKnownSignature method